### PR TITLE
fix our installation (instructions)

### PR DESCRIPTION
**Pull request by igalic**
* chore(build): install cython
* doc(build): most systems will need libgit2 libucl

### CODE REVIEW

**Comment by skarekrow** [comment]
While it LGTM, I do not really like that we have a hard dependency on libucl. As many users may not be coming from `iocage-legacy`'s `develop` branch, they wouldn't need it. It would be nice to make this optional.

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ SERVER=""
 
 install:
 	python3.6 -m ensurepip
-	pkg install -q -y libgit2
+	pkg install -q -y libgit2 cython
```
Comment (igalic) [inline Makefile]: this should be `pip3.6 install cython`

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ pip3.6 install libiocage
 ### from Master branch
 
 ```
+pkg install -y libgit2 libucl
```
Comment (igalic) [inline README.md]: this line shouldn't be necessary at all…

File: Makefile (hunk 1 above)
Comment (igalic) [inline Makefile]: instead, we need libucl here.

Comment (igalic) [inline Makefile]: we *could* install cython3 however, from pkg, it would make things a lot less painful.

**Comment by skarekrow** [comment]
Still LGTM

File: Makefile [hunk 3]
```diff
@@ -3,7 +3,7 @@ SERVER=""
 
 install:
 	python3.6 -m ensurepip
-	pkg install -q -y libgit2
+	pkg install -q -y libgit2 libucl cython3
```
Comment (igalic) [inline Makefile]: this works, but only if we don't use our own venv…
which, per our docs, we don't (recommend)

**Comment by gronke** [comment]
@skarekrow the hard dependency on libucl is only for developers. We can argue if we want to introduce configurations to enable or disable features, but it seems more acceptable to bundle libucl than making a special case out of it. Ah, btw. libucl is also used to parse rc.conf files 😏

**Comment by skarekrow** [comment]
@gronke fair enough. ;)

**Pull request merged**